Contributing/Issue_Priorities.md

```diff
@@ -0,0 +1,32 @@
+<!---
+Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+## AMP GitHub Issue Priorities
+The AMP team is introducing the below priorities and guidelines to make it easier to order issues by the level of attention they need. 
+These new guidelines can give you also a good overview of when to expect updates or closure of issues.
+
+In order to add a priority to your issues use our [list of GitHub labels](https://github.com/ampproject/amphtml/labels).
+
+### Priorities & Guidelines
+
+
+ Priority           | What it means          | Guidelines for Bugs              | Guidelines  for FRs               
+--------------------| ---------------------- | ---------------------------------| --------------------------------- 
+P0: Drop Everything | <ul><li>Outage</li><li>Critical production issue</li></ul>    | <ul><li>Drop everything until this is fixed</li><li>Should be assigned and accepted within 24 hours of filing</li><li>Provide regular bug update on status and ETA of the fix, once every day after acceptance</li><li>Fixed in 7 days (next release or patch release, whichever is sooner)</li></ul>  |  <ul><li>A FR is almost never a P0</li></ul>
+P1: High Priority   | <ul><li>Breakage of a critical feature or user journey</li><li>Highly significant feature</li></ul>     | <ul><li>Should be assigned and accepted within 48 hours of filing</li><li>Should be updated by the team once every week after acceptance</li><li>Fixed within 30 days (next release) or updated with explanation and timeline</li></ul> | <ul><li>Implementation guidelines: 2 weeks</li><li>Feature requests are not P1, unless a situation develops where the lack of the feature is significant user problem</li></ul>
```

Comment:
"Fixed within 30 days (next release)."  Are we saying "fixed in the next release, which will always be within 30 days?"  Or "fixed by the next release after 30 days."?  If the former I'd rephrase this as "fixed in the next release (within 30 days)."

For the FR part, "unless a situation develops where the lack of the feature is a significant user problem" is too ambiguous.  My sense is that it's quite common for someone filing a FR to feel like it's addressing a "significant user problem" so I think we need more stringent terminology here.